frontend/ui/elements/screensaver_menu.lua

```diff
@@ -13,6 +13,7 @@ end
 local function whiteBackground() return G_reader_settings:isTrue("screensaver_white_background") end
 local function noBackground() return G_reader_settings:isTrue("screensaver_no_background") end
 local function stretchImages() return G_reader_settings:isTrue("screensaver_stretch_images") end
+local function messagePos() return G_reader_settings:readSetting("screensaver_message_position") end
```

Comment:
No real reason to abbreviate this one :) those above are not. `messagePosition()` is not that long.